src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -794,18 +789,42 @@ private static void finish(@CheckForNull final String cookie) {
         private static final class Callback extends BodyExecutionCallback.TailCall {
```

Comment:
Do we need to add an explicit `serialVersionUID` to this class to avoid compatibility issues since a field has been added?

Reply:
That would probably make sense.